StortSpelProjekt/Vendor/config.txt

```diff
@@ -7,6 +7,11 @@ i_resolutionHeight 800
 i_windowWidth 1200
 i_windowHeight 800
 i_windowMode 0
+i_resolutionWidth 1920
```

Comment:
Why do you set these things twice?
Why do you set these things two times?

Reply:
Did not notice this good catch :), some merge went wrong.